sys/net/gnrc/netif2/gnrc_netif2.c

```diff
@@ -536,10 +536,30 @@ int gnrc_netif2_ipv6_addr_add(gnrc_netif2_t *netif, const ipv6_addr_t *addr,
     }
     netif->ipv6.addrs_flags[idx] = flags;
     memcpy(&netif->ipv6.addrs[idx], addr, sizeof(netif->ipv6.addrs[idx]));
-    /* TODO:
-     *  - update prefix list, if flags == VALID
-     *  - with SLAAC, send out NS otherwise for DAD probing */
+#ifdef MODULE_GNRC_IPV6_NIB
+    if (_get_state(netif, idx) == GNRC_NETIF2_IPV6_ADDRS_FLAGS_STATE_VALID) {
+        void *state;
```

Comment:
This looks like the cause of the segfault. Initializing the `void *state` to `NULL` fixes it.

Reply:
Yes that is required! Will fix. Thank you for investigating!

Reply:
Fixed.